src/calculateRentalCost.js

```diff
@@ -3,8 +3,30 @@
  *
  * @return {number}
  */
+const costRental = 40;
+const mediumTerm = 3;
+const mediumTermDiscound = 20;
+const longTerm = 7;
+const longTermDiscount = 50;
+
 function calculateRentalCost(days) {
-  // write code here
+  let result = 0;
+
+  if (days >= longTerm) {
+    result = days * costRental - longTermDiscount;
+
+    return result;
+  }
+
+  if (days >= mediumTerm) {
+    result = days * costRental - mediumTermDiscound;
```

Comment:
The variable `mediumTermDiscound` is misspelled. It should be `mediumTermDiscount` to correctly apply the medium-term discount.